ui/src/app/pages/data/data-set/export-data-set-modal.tsx

```diff
@@ -119,19 +119,28 @@ class ExportDataSetModal extends React.Component<
       cohortIds: dataSet.cohorts.map(c => c.id),
       domainValuePairs: dataSet.domainValuePairs
     };
-    await dataSetApi().exportToNotebook(
-      workspaceNamespace, workspaceFirecloudName,
-      {
-        dataSetRequest: request,
-        notebookName: this.state.notebookName,
-        newNotebook: this.state.newNotebook,
-        kernelType: this.state.kernelType
-      });
-    // Open notebook in a new tab and close the modal
-    const notebookUrl = `/workspaces/${workspaceNamespace}/${workspaceFirecloudName}/notebooks/
-      preview/${appendNotebookFileSuffix(encodeURIComponentStrict(this.state.notebookName))}`;
-    navigateByUrl(notebookUrl);
-    this.props.closeFunction();
+    try {
+      await dataSetApi().exportToNotebook(
+        workspaceNamespace, workspaceFirecloudName,
+        {
+          dataSetRequest: request,
+          notebookName: this.state.notebookName,
+          newNotebook: this.state.newNotebook,
+          kernelType: this.state.kernelType
+        });
+      // Open notebook in a new tab and close the modal
+      const notebookUrl = `/workspaces/${workspaceNamespace}/${workspaceFirecloudName}` +
+        `/notebooks/preview/${appendNotebookFileSuffix(
+          encodeURIComponentStrict(this.state.notebookName))}`;
+      navigateByUrl(notebookUrl);
+      this.props.closeFunction();
+    } catch (error) {
```

Comment:
This is a different bug: https://precisionmedicineinitiative.atlassian.net/browse/RW-3782 caused an infinite spinner in an uncancellable modal here.  The bug remains but now the user is not prevented from cancelling out of this modal and going back to other work